### PR TITLE
Homework 9

### PR DESCRIPTION
**Pull request by AbdurakhimovAziz**

### CODE REVIEW

File: client/src/app/auth/auth.guard.spec.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { TestBed } from '@angular/core/testing';
 
 import { AuthGuard } from './auth.guard';
 
-describe('AuthGuard', () => {
+xdescribe('AuthGuard', () => {
```
Comment (AnastasiaHarost) [inline client/src/app/auth/auth.guard.spec.ts:5]: why do you use xdescribe?

Comment (AbdurakhimovAziz) [reply client/src/app/auth/auth.guard.spec.ts:5]: to disable unnecessary tests, wrote tests only for one service. Should I delete  other files instead?

Comment (AnastasiaHarost) [reply client/src/app/auth/auth.guard.spec.ts:5]: it's okey. Sorry, forgot about your comment

**Review by AnastasiaHarost — APPROVED** [review]
Thank you for you work, good luck on the interview:)

**Comment by AbdurakhimovAziz** [comment]
> Thank you for you work, good luck on the interview:)

Thank you very much)

**Pull request merged**